common/gamelib/move-check.mjs

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 HITCON Online Contributors
+// SPDX-License-Identifier: BSD-2-Clause
+
+const PLAYER_MOVE_TIME_INTERVAL = 250; // ms
+const PLAYER_MOVE_DISTANCE_LIMIT = 1; // cell
+
+/**
+ * TODO
+ * @param {Player} oldPlayerData - TODO
+ * @param {PlayerSyncMessage} newPlayerData - TODO
+ * @param {GameMap} gameMap - TODO
+ * @return {Boolean}
+ */
+function checkPlayerMove(oldPlayerData, newPlayerData, gameMap) {
```

Comment:
It seems that `newPlayerData` is `PlayerSyncMessage` type while `oldPlayerData` is `Player` type. Maybe we can rename the variable for better readability?

Reply:
fixed